docs/docs/guides/resource-manager.mdx

```diff
@@ -0,0 +1,269 @@
+---
+title: Using Resource Managers
+---
+
+The goal of this guide is to show you how to create a resource pool, and how you can allocated resources with them.
+
+:::note
+
+The guide makes the assumption that we start with an Infrahub instance that doesn't have any data or schema loaded.
+
+:::
+
+## Loading a schema
+
+Save the following schema in a file on your local system. The location or filename are not that important, but in this guide will be using `/tmp/schema.yml`
+
+```yaml
+---
+version: "1.0"
+nodes:
+  - name: IPPrefix
+    namespace: Ipam
+    include_in_menu: false
+    inherit_from:
+      - "BuiltinIPPrefix"
+    description: "IPv4 or IPv6 network"
+    label: "IP Prefix"
+  - name: IPAddress
+    namespace: Ipam
+    include_in_menu: false
+    inherit_from:
+      - "BuiltinIPAddress"
+    description: "IP Address"
+    label: "IP Address"
+  - name: Device
+    namespace: Infra
+    description: "A Device"
+    icon: "mdi:server"
+    label: "Device"
+    attributes:
+      - name: name
+        kind: Text
+        label: Name
+        optional: false
+    relationships:
+      - name: primary_ip
+        label: "Primary IP Address"
+        peer: IpamIPAddress
+        optional: false
+        kind: Attribute
+        cardinality: one
+```
```

Comment:
Perhaps add a "note" or "warning" that this schema might conflict with the other demo schema if that has been loaded?

Reply:
The guide mentions this near the top of the page
![image](https://github.com/opsmill/infrahub/assets/7521270/e657ae01-edf3-4fa3-8af1-9a331abbddbd)